scripting/pugsetup.sp

```diff
@@ -2237,3 +2244,30 @@ stock bool PermissionFromString(const char[] permissionString, Permission& p,
 
   return true;
 }
+
+public int GetTotalPlayers()
+{
+    int count;
+    LoopClients(client)
```

Comment:
Use full code remove macro

Reply:
why? doesnt make difference. This way is better to read too.

Reply:
I would prefer not introducing this macro. Such macros *can* aid readability, but I don't think it's worth it for this single usage.  

Additionally, the macros is making it harder to see a fairly obvious bug here: isn't this function always returning `MaxClients`? There's no conditional in the loop, `count` will simply be increment `MaxClients` times.

Reply:
Alright, removed macro. Can you review it again?